.openshift/rocket-chat-ephemeral.json

```diff
@@ -262,6 +262,30 @@
                                         "mountPath": "/app/uploads"
                                     }
                                 ],
+                                "readinessProbe": {
+                                    "httpGet": {
+                                        "path": "/api/v1/info",
+                                        "port": 3000,
+                                        "scheme": "HTTP"
+                                    },
+                                    "initialDelaySeconds": 5,
+                                    "timeoutSeconds": 1,
+                                    "periodSeconds": 10,
+                                    "successThreshold": 1,
+                                    "failureThreshold": 3
```

Comment:
@jfchevrette did a little verifying.  So it does not wait on readiness check to pass.  Which is a bit unfortunate.  In the future might be able to substitute this for a command in the container that would know that it was still in the process of starting.  I'll go ahead and approve this and we can tweak this as time goes on based on feedback.  

Thanks!